activesupport/lib/active_support/core_ext/kernel/reporting.rb

```diff
@@ -1,6 +1,9 @@
 module Kernel
   module_function
 
+  DEFAULT_SUPPRESS_EXCEPTIONS = [StandardError].freeze
```

Comment:
Idea was to disallow mutation of that list.
Although now I believe there's no need in that constant at all:

``` ruby
def suppress(*exceptions_list)
  exceptions_list = StandardError if exceptions_list.empty?
  # ...
end
```